configuration/backend_configuration/ampathforms/procedure-history-form.json

```diff
@@ -0,0 +1,225 @@
+                    "label": "Minutes"
+                  },
+                  {
+                    "concept": "1822AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Hours"
+                  },
+                  {
+                    "concept": "1072AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Days"
+                  },
+                  {
+                    "concept": "1073AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Weeks"
+                  },
+                  {
+                    "concept": "1074AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Months"
+                  },
+                  {
+                    "concept": "1734AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Years"
+                  }
+                ]
+              },
+              "clearWhenExpression": "Number(procedureDuration) > 99 && (procedureDurationUnit === '1734AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA')",
+              "validators": [
+                {
+                  "type": "js_expression",
+                  "failsWhenExpression": "procedureDurationUnit === '1734AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' && Number(procedureDuration) > 99",
+                  "message": "Duration cannot exceed 100 years.. Change unit or reduce duration."
+                },
+                {
+                  "type": "js_expression",
+                  "failsWhenExpression": "procedureDurationUnit === '1733AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' && Number(procedureDuration) > 1440",
+                  "message": "Duration cannot exceed 1440 minutes.. Change unit or reduce duration."
+                },
+                {
+                  "type": "js_expression",
+                  "failsWhenExpression": "procedureDurationUnit === '1822AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' && Number(procedureDuration) > 168",
+                  "message": "Duration cannot exceed 168 hours.. Change unit or reduce duration."
+                },
+                {
+                  "type": "js_expression",
+                  "failsWhenExpression": "procedureDurationUnit === '1072AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' && Number(procedureDuration) > 365",
+                  "message": "Duration cannot exceed 365 days.. Change unit or reduce duration."
+                },
+                {
+                  "type": "js_expression",
+                  "failsWhenExpression": "procedureDurationUnit === '1073AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' && Number(procedureDuration) > 52",
+                  "message": "Duration cannot exceed 52 weeks.. Change unit or reduce duration."
+                },
+                {
+                  "type": "js_expression",
+                  "failsWhenExpression": "procedureDurationUnit === '1074AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' && Number(procedureDuration) > 12",
+                  "message": "Duration cannot exceed 12 months.. Change unit or reduce duration."
+                }
```

Comment:
I'm not sure we have a good reason for allowing durations in weeks, months or years?